src/Microsoft.AspNet.WebUtilities/BufferedReadStream.cs

```diff
@@ -315,38 +315,42 @@ public bool EnsureBuffered(int minCount)
         public string ReadLine(int lengthLimit)
         {
             CheckDisposed();
-            var builder = new MemoryStream(200);
-            bool foundCR = false, foundCRLF = false;
-
-            while (!foundCRLF && EnsureBuffered())
+            using (var builder = new MemoryStream(200))
```

Comment:
MemoryStream.Dispose doesn't really do anything. Are you calling it just as a best practice?